test_app/migrations/0004_remove_encryptionmodel_created_on_and_more.py

```diff
@@ -0,0 +1,109 @@
+            model_name='team',
+            name='created_on',
+        ),
+        migrations.RemoveField(
+            model_name='team',
+            name='modified_on',
+        ),
+        migrations.RemoveField(
+            model_name='user',
+            name='created_on',
+        ),
+        migrations.RemoveField(
+            model_name='user',
+            name='modified_on',
+        ),
+        migrations.AddField(
+            model_name='encryptionmodel',
+            name='created',
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now, help_text='The date/time this resource was created'),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='encryptionmodel',
+            name='modified',
+            field=models.DateTimeField(auto_now=True, help_text='The date/time this resource was created'),
+        ),
+        migrations.AddField(
+            model_name='organization',
+            name='created',
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now, help_text='The date/time this resource was created'),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='organization',
+            name='modified',
+            field=models.DateTimeField(auto_now=True, help_text='The date/time this resource was created'),
+        ),
+        migrations.AddField(
+            model_name='relatedfieldstestmodel',
+            name='created',
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now, help_text='The date/time this resource was created'),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='relatedfieldstestmodel',
+            name='modified',
+            field=models.DateTimeField(auto_now=True, help_text='The date/time this resource was created'),
+        ),
+        migrations.AddField(
+            model_name='team',
+            name='created',
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now, help_text='The date/time this resource was created'),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='team',
+            name='modified',
+            field=models.DateTimeField(auto_now=True, help_text='The date/time this resource was created'),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='created',
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now, help_text='The date/time this resource was created'),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='modified',
+            field=models.DateTimeField(auto_now=True, help_text='The date/time this resource was created'),
```

Comment:
I would, personally, find this change easier to manage if just thrown in the 0001 migration. After this merges, I have to rebase the RBAC PR and do the replacements there. By having this migration, I have to tease out what fields in the 0001 migration are _supposed_ to be `created_on` to later be changed, and which are new models and need to have the new fields.

Reply:
I don't think this is a good idea, since other projects like EDA and Hub are relying on DAB already. It will make migration painful and has to be coordinated with projects, that depend on DAB.

Reply:
Product wise if AWX and EDA have not yet shipped with any models that would be affected by this it would be fine to merge into 0001. But this would also require any dev environments to roll back to zero and then roll forward or their environments would be "stuck", right? If those statements are both accurate, are we ok with that?

Reply:
This is test_app. My problems resolving conflicts on rebase are 80% in test_app.

You could repeat this discussion for the authenticator app, but that doesn't affect my own workload.

It would also strike me as poor form to have an initial release of a server product containing a dozen or more migrations, but that's also not immediately relevant to me right now.

Reply:
Thats fair for test app, its just a playground. Should we squash all of the migrations or just merge this new one into the older ones?

Reply:
the 0002 migration in test_app has a technical reason for its existence. The rest do not.

Reply:
Oh, thats interesting, IDK if you can "squash with skip". I'll probably have to manually merge them.